sonar-ballerina-plugin/src/main/resources/io/ballerina/sonar/ballerina-3.html

```diff
@@ -0,0 +1,31 @@
+<p>
+    A call to an isolated function is concurrency-safe if it is called with arguments that are safe at least until the call returns.
```

Comment:
This rule is defined for public constructs only

Reply:
Resolved with f10b04b7337bff186e2de3ae2a65a2421c10884d